test/integrations/routes/articles.test.js

```diff
@@ -95,188 +95,6 @@ describe('API endpoint: /api/articles (Routes)', () => {
-          expect(res.body.data.page).to.haveOwnProperty('last').to.be.a('number');
-          done();
-        });
-    });
-    it('Should return all articles in that page if page parameter is valid', (done) => {
-      chai
-        .request(app)
-        .get('/api/v1/articles')
-        .query({ page: 1 })
-        .end((err, res) => {
-          expect(res).to.have.status(STATUS.OK);
-          expect(res.body).to.be.an('object');
-          expect(res.body).to.haveOwnProperty('code').equal(STATUS.OK);
-          expect(res.body).to.haveOwnProperty('message').equal(MESSAGE.ARTICLES_FOUND);
-          expect(res.body).to.haveOwnProperty('status').to.equal(true);
-          expect(res.body).to.haveOwnProperty('data').to.be.an('object');
-          expect(res.body.data).to.haveOwnProperty('articles').to.be.an('array');
-          expect(res.body.data).to.haveOwnProperty('page');
-          expect(res.body.data.page).to.haveOwnProperty('first').to.equal(1);
-          expect(res.body.data.page).to.haveOwnProperty('current').to.equal(1);
-          expect(res.body.data.page).to.haveOwnProperty('last').to.be.a('number');
-          done();
-        });
-    });
-    it(`Should return first ${PAGE_LIMIT} articles if page parameter is a negetive number`, (done) => {
-      chai
-        .request(app)
-        .get('/api/v1/articles')
-        .query({ page: -2 })
-        .end((err, res) => {
-          expect(res).to.have.status(STATUS.OK);
-          expect(res.body).to.be.an('object');
-          expect(res.body).to.haveOwnProperty('code').to.equal(STATUS.OK);
-          expect(res.body).to.haveOwnProperty('message').equal(MESSAGE.ARTICLES_FOUND);
-          expect(res.body).to.haveOwnProperty('status').to.equal(true);
-          expect(res.body).to.haveOwnProperty('data').to.be.an('object');
-          expect(res.body.data).to.haveOwnProperty('articles').to.be.an('array');
-          expect(res.body.data).to.haveOwnProperty('page');
-          expect(res.body.data.page).to.haveOwnProperty('first').to.equal(1);
-          expect(res.body.data.page).to.haveOwnProperty('current').to.equal(1);
-          expect(res.body.data.page).to.haveOwnProperty('last').to.be.a('number');
-          done();
-        });
-    });
-    it('Should not return any articles if page parameter is larger than available articles', (done) => {
-      chai
-        .request(app)
-        .get('/api/v1/articles')
-        .query({ page: 100 })
-        .end((err, res) => {
-          expect(res).to.have.status(STATUS.NOT_FOUND);
-          expect(res.body).to.be.an('object');
-          expect(res.body).to.haveOwnProperty('code').to.equal(STATUS.NOT_FOUND);
-          expect(res.body).to.haveOwnProperty('message').equal(MESSAGE.ARTICLES_NOT_FOUND);
-          expect(res.body).to.haveOwnProperty('status').to.equal(false);
-          expect(res.body).to.haveOwnProperty('data').to.be.an('array');
-          done();
-        });
-    });
-  });
-
```

Comment:
These tests have been deleted. Is the PR a WIP PR?

Reply:
Yes @thislekan, the PR is still a work in progress.